Tests/SonarScanner.MSBuild.PreProcessor.Test/WebClientDownloaderBuilderTest.cs

```diff
@@ -377,11 +364,12 @@ public async Task SelfSignedServerCertificate_IgnoredIfServerIsTrusted()
         };
 
         // Act
-        var result = await client.Download("https://httpbin.org/user-agent");
+        var result = await client.Download("https://www.cloudflarestatus.com/api/v2/status.json");
```

Comment:
Cloudflare is the backbone of the internet and `cloudflarestatus.com/` should have almost 100% uptime.